etc/profile-m-z/ping.profile

```diff
@@ -7,52 +7,65 @@ include ping.local
 # Persistent global definitions
 include globals.local
 
-blacklist /tmp/.X11-unix
 blacklist ${RUNUSER}
 
 include disable-common.inc
 include disable-devel.inc
 include disable-exec.inc
 include disable-interpreters.inc
+include disable-proc.inc
 include disable-programs.inc
+include disable-X11.inc
 include disable-xdg.inc
 
 include whitelist-common.inc
+include whitelist-run-common.inc
+include whitelist-runuser-common.inc
 include whitelist-usr-share-common.inc
 include whitelist-var-common.inc
 
+# Add the next line to your ping.local if your kernel allows unprivileged userns clone.
+include ping-hardened.inc.profile
```

Comment:
Did you mean?
```suggestion
#include ping-hardened.inc.profile
```

Reply:
Yeah, that's what I meant. Fixed, thanks.